test/test_cohort_defaults.py

```diff
@@ -32,7 +32,44 @@
 
 FILE_FORMAT_1 = "patient_format1_%s.vcf"
 
-def test_default_filter_fn():
+def test_default_filter_fn_with_variant_count():
```

Comment:
no, because one (`missense_snv_count`) will break without the 'closure' cache check & the other does not. it helps to have both tests in there to see where the failure occurs.